code/WorkInProgress/blueprints.dm

```diff
@@ -436,8 +436,8 @@
 
 	var/static/savefile/save = new/savefile("data/blueprints.dat")
 
-	afterattack(atom/target as mob|obj|turf, mob/user as mob)
-		if(GET_DIST(src,target) > 2) return
+	pixelaction(atom/target as mob|obj|turf, params, mob/user as mob)
+	//	if(GET_DIST(src,target) > 2) return
 
 		if(!isturf(target)) target = get_turf(target)
```

Comment:
```suggestion
	pixelaction(atom/target, params, mob/user)
		if(!isturf(target)) target = get_turf(target)
```
Remove the commented out code; Git history will remember.
I don't believe that there is any need for `as mob|obj|turf` here.